libraries/devkit_install.rb

```diff
@@ -0,0 +1,82 @@
+#
+# Cookbook Name:: languages
+# HWRP:: ruby_install
+#
+# Copyright 2015, Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+require 'chef_compat/resource'
+
+require_relative '_helper'
+require 'yaml'
+
+class Chef
+  class Resource::DevkitInstall < ChefCompat::Resource
```

Comment:
Why did we create a resource for this? A simple helper would suffice. I also want to stop the experimenting with `ChefCompat::Resource` it's not the right fit for this set of primitives.